subsys: Add network core monitor (#11979)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

This section provides detailed lists of changes by :ref:`library <libraries>`.

* Added :ref:`nrf_security` library, relocated from the sdk-nrfxlib repository to the :file:`subsys/nrf_security` directory.

> * Added:ref:`network_core_monitor` module to monitor the status of the nRF5340 processor's network core.

Reviewer: ```suggestion
* Added :ref:`network_core_monitor` library for monitoring the status of the nRF5340 processor's network core.
```
